### ISSUE TITLE
User Story - Accessibility

### ISSUE THREAD

**Issue by Jcasa08** [Jcasa08/prevent-aware#2]
As a user with accessibility needs, I want the website to have proper colour contrast and alt text for images, so that I can access all the information regardless of my abilities.

Acceptance Criteria:

- [x]  All images used should have alt text

- [x] Colour pallete used should contrast and be easy to read

Tasks:

- [x] Choose an appropriate colour pallete

- [x] Ensure all images have an alt text and approriate aria labels are used.

- [ ] Use W3C's accessibility checker to ensure accessability standards are met
